Formula/inotify-tools.rb

```diff
@@ -0,0 +1,31 @@
+class InotifyTools < Formula
+  desc "C library and command-line programs providing a simple interface to inotify"
+  homepage "https://github.com/inotify-tools/inotify-tools"
+  url "https://github.com/inotify-tools/inotify-tools/archive/refs/tags/3.22.1.0.tar.gz"
+  sha256 "da81010756866966e6dfb1521c2be2f0946e7626fa29122e1672dc654fc89ff3"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on :linux
+
+  def install
+    inreplace "libinotifytools/src/inotifytools.c", "#ifdef __FreeBSD__", "#ifdef __APPLE__"
```

Comment:
This seems like a strange `inreplace` for a Linux-only formula

Reply:
Ha ha yes let me remove it, I was trying to fix it on macOS.
That inreplace helped, I was able to compile it, but it was not usable on macOS.
I did not have time to investigate more, so I made the formula linux only.

Looks like upstream is active so they might fix it on macOS one day.